### ISSUE TITLE
Exec format error for arm64 package jq 1.6.0-2

### ISSUE THREAD

**Issue by BFarand** [cloudposse/packages#4611]
Found a bug? Maybe our [Slack Community](https://slack.cloudposse.com) can help.

## Describe the Bug

When attempting to use `jq:1.6.0-2` on the arm64 architecture, the following error is reported:

```shell
ubuntu@example-xyz:~$ jq
bash: /usr/bin/jq: cannot execute binary file: Exec format error
```

This is due to the provided package actually being for the x86_64 architecture.

```shell
ubuntu@example-xyz:~$ file /usr/bin/jq
/usr/bin/jq: ELF 64-bit LSB executable, x86-64, version 1 (GNU/Linux), statically linked, for GNU/Linux 2.6.32, BuildID[sha1]=c159e8a134ee222019dfbac0a1af4bc94b302eb0, with debug_info, not stripped
```

## Expected Behavior

```shell
ubuntu@example-xyz:~$ jq
jq - commandline JSON processor [version 1.6]
 
Usage: jq [options] [file...]
        jq [options] --args [strings...]
        jq [options] --jsonargs [JSON_TEXTS...]
 
jq is a tool for processing JSON inputs, applying the given filter to
its JSON text inputs and producing the filter's results as JSON on
standard output.
 
The simplest filter is ., which copies jq's input to its output
unmodified (except for formatting, but note that IEEE754 is used
for number representation internally, with all that that implies).
 
For more advanced filters see the jq(1) manpage ("man jq")
and/or https://stedolan.github.io/jq
 
Example:
 
        $ echo '{"foo": 0}' | jq .
        {
                "foo": 0
        }
 
For a listing of options, use jq --help.
```

```shell
ubuntu@example-xyz:~$ file /usr/bin/jq 
/usr/bin/jq: ELF 64-bit LSB pie executable, ARM aarch64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, BuildID[sha1]=57470ed34b63415bc7afc5a234741f0f38d2649b, for GNU/Linux 3.7.0, stripped
```

(noteworthy thing being the **ARM aarch64** instead of **x86-64**)

## Steps to Reproduce

1. Add the cloudsmith cloudposse repository
   a. `curl -1sLf 'https://dl.cloudsmith.io/public/cloudposse/packages/setup.deb.sh' | sudo -E bash`.
2. Install jq (from the above repository).
   a. `sudo apt-get install -y jq`.
3. Attempt to execute `jq`.

See description and expected behaviour for more 'complete' details.

## Environment (please complete the following information):

```shell
ubuntu@example-xyz:~$ cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.4 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.4 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

```shell
ubuntu@example-xyz:~$ uname -m
aarch64
```

**Comment by Nuru** [comment]
@BFarand Thank you very much for bringing this to our attention. 

tl;dr: The issue has been resolved.

### The long story

When we first started publishing a `jq` package, the tool had not been widely adopted and we published the package because none of the mainstream repos were publishing packages for it. 

We only added support for ARM/arm64/aarch64 relatively recently. At the time `jq` 1.6 was first released, the published release did not include an ARM binary. However, by then, mainstream package repos were publishing `jq`, and included an ARM binary for v1.6, so we decided to stop publishing the `jq` package ourselves.

Unfortunately, in the process leading up to that decision, going through some attempts to publish our own ARM package, we released package version `1.6.0-r2`, which has semantic version priority over version `1.6-*` and `1.6.0-r1` which is what the mainstream repos publish, and our ARM package has, as you noted, an x86_64 binary, which is obviously wrong but passed our testing at the time (which was still quite new regarding ARM binaries). 

Thanks to your report, we have now removed the `jq v1.6*` Debian and RPM packages from our repo, ensuring you can install versions blessed and maintained by the OS maintainers. (We have left our Alpine `jq v1.6` packages in place, since they only ever supported `x86_64`, but Alpine now, too, has mainstream packages for `jq`.)

```
sudo apt-get install -y jq
```

should now get you the latest supported version of `jq` with the correct architecture, whether or not you have the Cloud Posse package repo installed. 

CC: @osterman @aknysh
